### ISSUE TITLE
train.lua in cpu mode complains about a nil value

### ISSUE THREAD

**Issue by imanthos** [jcjohnson/densecap#67]
Hello, 

After running the **_th train.lua -gpu -1_** command it shows the following output:

loading network weights from .. VGG_ILSVRC_16_layers.caffemodel	
**Couldn't load /home/sentinel/5G - Project/densecap/data/models/vgg-16/VGG_ILSVRC_16_layers.caffemodel
/home/sentinel/torch/install/bin/luajit: ./densecap/net_utils.lua:100: attempt to index local 'net' (a nil value)**
_stack traceback:
	./densecap/net_utils.lua:100: in function 'subsequence'
	./densecap/DenseCapModel.lua:73: in function '__init'
	/home/sentinel/torch/install/share/lua/5.1/torch/init.lua:91: in function </home/sentinel/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'DenseCapModel'
	./models.lua:7: in function 'setup'
	train.lua:48: in main chunk
	[C]: in function 'dofile'
	...inel/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00406670_

Also running the same command with some flags more:
**th train.lua -gpu -1 -backend nn -checkpoint_start_from data/models/densecap/model_id1-501-1448236541.t7_cpu.t7**, and another model breaks with the same cause:

**/home/sentinel/torch/install/bin/luajit: ./models.lua:11: attempt to index local 'model' (a nil value)**
_stack traceback:
	./models.lua:11: in function 'setup'
	train.lua:48: in main chunk
	[C]: in function 'dofile'
	...inel/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00406670_

Is this a dependency issue or something I am missing? 

Also sharing the rest of the output in case it is helpful:
{
  finetune_cnn_after : -1
  weight_decay : 1e-06
  optim_beta2 : 0.999
  val_images_use : 1000
  timing : false
  optim_epsilon : 1e-08
  test_num_proposals : 1000
  input_encoding_size : 512
  losses_log_every : 10
  id : ""
  gpu : -1
  drop_prob : 0.5
  data_json : "data/VG-regions-dicts.json"
  clip_final_boxes : 1
  sampler_batch_size : 256
  rnn_size : 512
  test_rpn_nms_thresh : 0.7
  progress_dump_every : 100
  optim_beta1 : 0.9
  end_objectness_weight : 0.1
  seed : 123
  test_final_nms_thresh : 0.3
  eval_first_iteration : 0
  captioning_weight : 1
  checkpoint_start_from : ""
  sampler_high_thresh : 0.7
  mid_objectness_weight : 0.1
  data_h5 : "data/VG-regions.h5"
  train_remove_outbounds_boxes : 1
  sampler_low_thresh : 0.3
  debug_max_train_images : -1
  learning_rate : 1e-05
  rpn_hidden_dim : 512
  max_iters : -1
  checkpoint_path : "checkpoint.t7"
  save_checkpoint_every : 10000
  end_box_reg_weight : 0.1
  proposal_regions_h5 : ""
  backend : "nn"
  mid_box_reg_weight : 0.05
  box_reg_decay : 5e-05
}

Thank you.

**Comment by csnemes2** [comment]
just donwload http://www.robots.ox.ac.uk/%7Evgg/software/very_deep/caffe/VGG_ILSVRC_16_layers.caffemodel to the proper folder, namely:  Project/densecap/data/models/vgg-16/VGG_ILSVRC_16_layers.caffemodel
